packages/chain-adapters/src/utxo/UTXOBaseAdapter.ts

```diff
@@ -26,11 +35,18 @@ import {
   accountTypeToScriptType,
   chainIdToChainLabel,
   convertXpubVersion,
+  getStatus,
+  getType,
+  toPath,
   toRootDerivationPath
 } from '../utils'
 import { bnOrZero } from '../utils/bignumber'
+import { GetAddressInput } from './types'
+import { utxoSelect } from './utxoSelect'
+
+export const UTXOChainIds = [KnownChainIds.BitcoinMainnet, KnownChainIds.DogecoinMainnet] as const
 
-export type UTXOChainIds = KnownChainIds.BitcoinMainnet // to be extended in the future to include other UTXOs
+export type UtxoChainId = typeof UTXOChainIds[number]
```

Comment:
Differs from the cosmos-sdk/evm pattern/naming

Reply:
I had it as `type`:
https://github.com/shapeshift/lib/pull/888/commits/49ba780fa5b211500f7aec76fe7b8186454354bb#diff-91f7561fb5e5062d3ca7bc6c42e96c285c398a4518d8485bdcb9f672cb4148d7R47

Reply:
haha, yeah don't really understand the rationale here considering the array is never used as far as I can tell.

Reply:
@asamere @kaladinlight this was required for the safety in the constructor here https://github.com/shapeshift/lib/pull/888/commits/49ba780fa5b211500f7aec76fe7b8186454354bb#diff-f2eb75fa1da8667a2030c66b92bc981a07b98a53045439f8a9fc696aa60860b2R36

Reply:
ok, we can leave as is for now then. we have the `supportedChainIds` array as specified by the concrete implementation already that should be sufficient ultimately